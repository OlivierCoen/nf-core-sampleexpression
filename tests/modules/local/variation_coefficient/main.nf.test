nextflow_process {

    name "Test Process VARIATION_COEFFICIENT"
    script "modules/local/variation_coefficient/main.nf"
    process "VARIATION_COEFFICIENT"
    tag "var_coeff"

    test("Should run without failures") {

        when {

            process {
                """
                input[0] = file('$baseDir/tests/input/variation_coefficient/all_counts.log_cpm.csv')
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
