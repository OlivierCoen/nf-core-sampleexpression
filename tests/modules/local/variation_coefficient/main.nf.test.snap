{
    "Should run without failures": {
        "content": [
            {
                "0": [
                    "variation_coefficients.csv:md5,dd0954838e602d495590f816412ab8e8"
                ],
                "1": [
                    [
                        "VARIATION_COEFFICIENT",
                        "R",
                        "[1] \"R version 4.3.3 (2024-02-29)\""
                    ]
                ],
                "csv": [
                    "variation_coefficients.csv:md5,dd0954838e602d495590f816412ab8e8"
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "24.04.4"
        },
        "timestamp": "2024-11-02T11:44:16.501321914"
    }
}